src/lib/mongoBackend/MongoCommonUpdate.cpp

```diff
@@ -1566,8 +1566,10 @@ static void setResponseMetadata(ContextAttribute* caReq, ContextAttribute* caRes
     caRes->metadataVector.push_back(md);
   }
 
-  if (caReq->getLocation().length() > 0)
+  if ((caReq->getLocation().length() > 0) && (caReq->type != GEO_POINT))
   {
+    /* Note that if attribute type is geo:point then the user is using the "new way"
+     * of locating entities in NGSIv1, thus location metadata is not rendered */
```

Comment:
Not really... "new way" referes to geo:point as the new way (after Orion 1.1.0) of specifying entity location.

NTC